.github/workflows/pnp-input-api-build.yml

```diff
@@ -374,11 +401,11 @@ jobs:
             Build failed on ${{ github.event_name }} event. Workflow: ${{ github.workflow }}. Job: ${{github.job}}. Run id: <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.run_id }}>
           channel: ${{ secrets.slack-channel }}
           service-account-key: ${{ secrets.SECRET_AUTH }}
-      
+
   release:
     if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
-    needs: acceptance
+    needs: [staging, acceptance]
```

Comment:
Why did you add dependency to staging job here?

Reply:
Thought it is a good idea. Actually, the acceptance step already requires staging, so maybe it is not really needed. If so, then should I also remove this for Query API build workflow?

Reply:
Yes, you are right, `acceptance` job has already dependency to `staging` job that's why it's not needed for `release` job. In the workflow for Query APIs we might not have acceptance tests that's why we need to have dependency to both `staging` and `acceptance` jobs.